### PR TITLE
Ddpb 2131 money out category prof only

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: src/AppBundle/Controller/Report/MoneyInController.php [hunk 1]
```diff
@@ -87,10 +87,8 @@ public function stepAction(Request $request, $reportId, $step, $transactionId =
             [
                 'step' => $step,
                 'type'             => 'in',
-                'translator'       => $this->get('translator'),
-                'clientFirstName'  => $report->getClient()->getFirstname(),
                 'selectedCategory' => $transaction->getCategory(),
-                'user' => $this->getUser(),
```
Comment (elvisciotti) [inline src/AppBundle/Controller/Report/MoneyInController.php]: not used anymore

File: src/AppBundle/Controller/Report/MoneyOutController.php [hunk 2]
```diff
@@ -82,10 +82,8 @@ public function stepAction(Request $request, $reportId, $step, $transactionId =
         $form = $this->createForm(FormDir\Report\MoneyTransactionType::class, $transaction, [
             'step' => $step,
             'type'             => 'out',
-            'translator'       => $this->get('translator'),
```
Comment (elvisciotti) [inline src/AppBundle/Controller/Report/MoneyOutController.php]: not used anymore

File: src/AppBundle/Controller/Report/MoneyOutController.php [hunk 3]
```diff
@@ -82,10 +82,8 @@ public function stepAction(Request $request, $reportId, $step, $transactionId =
...
-            'clientFirstName'  => $report->getClient()->getFirstname(),
```
Comment (elvisciotti) [inline src/AppBundle/Controller/Report/MoneyOutController.php]: not used anymore

File: src/AppBundle/Controller/Report/MoneyOutController.php (hunk 2 above)
Comment (slizzio) [reply src/AppBundle/Controller/Report/MoneyOutController.php]: Can you confirm that this is not used to translate the categories in the form? Also there are references to the translator within the configureOptions (            ->setAllowedTypes('translator', TranslatorInterface::class);
)

File: tests/behat/features/prof/03-report/03-fees.feature [hunk 4]
```diff
@@ -1,123 +0,0 @@
...
-#     # change to yes
-#    When I click on "edit-exist" in the "has-fees" region
-#    And the step with the following values CAN be submitted:
-#      | prof_service_fees_currentProfPaymentsReceived_1 | no |
-#    And I should see "No" in the "has-fees" region
-#    And I should not see the "assessed-fee-annual-management-interim" region
-#    And I should not see the "previous-estimates" region
-#    And I should not see the "scco-reason" region
-#    # restore data before scenario
-#    And I load the application status from "prof-fees-expenses-before-edit"
```
Comment (slizzio) [inline tests/behat/features/prof/03-report/03-fees.feature]: Why is this commented out? Has it been replaced?

Comment (slizzio) [reply tests/behat/features/prof/03-report/03-fees.feature]: Ah i see, it was already commented out and its just been removed is that correct?

File: src/AppBundle/Controller/Report/MoneyOutController.php (hunk 2 above)
Comment (elvisciotti) [reply src/AppBundle/Controller/Report/MoneyOutController.php]: the "translator" in that form is removed completely along with the "setAllowedTypes". 
I think we can consider deprecated the usage of the translator inside the form, as symfony (and our custom twig helpers) already do the job of translating once the key is given, and the translation_domain is set in the form

File: tests/behat/features/prof/03-report/03-fees.feature (hunk 4 above)
Comment (elvisciotti) [reply tests/behat/features/prof/03-report/03-fees.feature]: yes, I've just removed it

**Pull request merged**